.travis.yml

```diff
@@ -1,14 +1,28 @@
 language: php
 
-php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - hhvm
+matrix:
+    include:
+        - php: 5.3
+        - php: 5.4
+        - php: 5.5
+        - php: 5.6
```

Comment:
I suggest keeping these builds in the ``php`` section instead of including them explicitly in the matrix here. It would be easier to understand for people reading this file by matching the common cases (Symfony does it differently only because @nicolas-grekas tried to run the component builds first at one point, which required building the whole matrix manually)